sdk/programmableconnectivity/azure-programmableconnectivity/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 1.0.0b1 (2024-03-27)
```

Comment:
The CI will fail if this date is in the past and it doesn't find a published release. Suggest changing this to Unreleased and adding a date later when you know the date you'll be releasing

Reply:
ok, will change to unreleased